config_bucket.go

```diff
@@ -11,32 +11,45 @@ import (
 )
 
 const (
-	BucketTypeLocal = "local"
-	BucketTypeS3    = "s3"
+	BucketTypeGridFS = "gridfs"
+	BucketTypeLocal  = "local"
+	BucketTypeS3     = "s3"
 )
 
-// BucketConfig represents the admin config section for bucket storage.
+// BucketsConfig represents the admin config section for bucket storage.
```

Comment:
Since S3 is used in various contexts throughout Evergreen, can we make this a little more explicit and state that this is bucket storage for storing internally-owned Evergreen data (as opposed to other potential usages like public buckets that all users can directly store data in)?